polaris/polaris/utils.py

```diff
@@ -109,15 +109,42 @@ def verify_valid_asset_operation(
         )
 
 
-def create_stellar_deposit(
-    transaction: Transaction, destination_exists: bool = False
-) -> bool:
+def create_stellar_deposit(transaction: Transaction) -> bool:
     """
     Create and submit the Stellar transaction for the deposit.
 
-    The Transaction can be either `pending_anchor` if the task is called
-    from `poll_pending_deposits()` or `pending_trust` if called from the
-    `check_trustlines()`.
+    *The Transaction `pending_trust` when the task is called from `check_trustlines()`.*
+
+    If called from check_trustlines() this means the following flow occurred
+    - The deposit transaction was blocked from sending a payment since the account
+      didn't have a trustline
+      - The account also issued a deposit from a Wallet that did not provide a
+        "claimable_balance_supported" value POST req body (or stated it as False).
+    - The Transaction was in pending_trust limbo until the account set up a trustline
+    - Trustline is established for the asset and create_stellar_deposit is called from
+      check_trustlines.py 127,1: if create_stellar_deposit(transaction):
```

Comment:
Don't reference specific code lines in comments
```suggestion
      check_trustlines()
```

Reply:
There are a couple other places below too